pkg/services/notifications/mailer.go

```diff
@@ -41,9 +57,8 @@ func (ns *NotificationService) send(msg *Message) (int, error) {
 
 		m.SetBody("text/html", msg.Body)
 
-		e := dialer.DialAndSend(m)
-		if e != nil {
-			err = errutil.Wrapf(e, "Failed to send notification to email address: %s", address)
+		if e := dialer.DialAndSend(m); err != nil {
```

Comment:
Shouldn't this be `if e := dialer.DialAndSend(m); e != nil`?

Reply:
Good catch. WIll fix